src/Entity/Timeline/Manifesto.php

```diff
@@ -0,0 +1,101 @@
+<?php
+
+namespace AppBundle\Entity\Timeline;
+
+use A2lix\I18nDoctrineBundle\Doctrine\ORM\Util\Translatable;
+use Algolia\AlgoliaSearchBundle\Mapping\Annotation as Algolia;
+use AppBundle\Entity\AbstractTranslatableEntity;
+use AppBundle\Entity\AlgoliaIndexedEntityInterface;
+use AppBundle\Entity\EntityMediaTrait;
+use Doctrine\Common\Collections\ArrayCollection;
+use Doctrine\ORM\Mapping as ORM;
+
+/**
+ * @ORM\Table(name="timeline_manifestos")
+ * @ORM\Entity(repositoryClass="AppBundle\Repository\Timeline\ManifestoRepository")
+ *
+ * @Algolia\Index(autoIndex=false)
+ */
+class Manifesto extends AbstractTranslatableEntity implements AlgoliaIndexedEntityInterface
+{
+    use EntityMediaTrait;
+    use Translatable;
+
+    /**
+     * @var int
+     *
+     * @ORM\Column(type="bigint")
+     * @ORM\Id
+     * @ORM\GeneratedValue
+     *
+     * @Algolia\Attribute
+     */
+    private $id;
+
+    public function __construct()
+    {
+        $this->translations = new ArrayCollection();
+    }
+
+    public function __toString()
+    {
+        /** @var ProfileTranslation $translation */
+        if ($translation = $this->translate()) {
+            return $translation->getTitle();
+        }
+
+        return '';
+    }
+
+    public function getId(): ?int
+    {
+        return $this->id;
+    }
+
+    /**
+     * @Algolia\Attribute(algoliaName="image")
+     */
+    public function getImage(): ?string
+    {
+        return $this->media ? $this->media->getPathWithDirectory() : null;
+    }
+
+    /**
+     * @Algolia\Attribute(algoliaName="titles")
+     */
+    public function getTitles(): array
+    {
+        return $this->getFieldTranslations('title');
+    }
+
+    /**
+     * @Algolia\Attribute(algoliaName="slugs")
+     */
+    public function getSlugs(): array
+    {
+        return $this->getFieldTranslations('slug');
+    }
+
+    /**
+     * @Algolia\Attribute(algoliaName="descriptions")
+     */
+    public function getDescriptions(): array
+    {
+        return $this->getFieldTranslations('description');
+    }
+
+    public function exportTitles(): string
+    {
+        return join(', ', $this->getTitles());
```

Comment:
What about using `implode` directly as `join` is his alias (https://www.php.net/manual/en/function.join.php) ?